### ISSUE TITLE
Integrate live search

### ISSUE THREAD

**Issue by ValiHar** [FHIR-Vaccination-Pass/patient-dashboard#29]
Integrate a live search mechanism for searching vaccine informations, appointments and past vaccinations. 

(This issue is blocked by the vaccination-server support for search)

**Comment by n-krueger** [comment]
Do you have a user story for this? E.g. what a user would want to search for in the patient dashboard.

**Comment by ValiHar** [comment]
Conclusion of last meeting: This feature is not planned and postponed for now.
